client/my-sites/posts/post-share.jsx

```diff
@@ -146,7 +146,7 @@ const PostSharing = React.createClass( {
 
 		return (
 			<div className="posts__post-share-wrapper">
-				{ this.props.requesting && <Notice status="is-warning" showDismiss={ false }>{ this.translate( 'Sharing...' ) }</Notice> }
+				{ this.props.requesting && <Notice status="is-warning" showDismiss={ false }>{ this.translate( 'Sharing…' ) }</Notice> }
```

Comment:
:disappointed: 20 existing translations will be lost with this change.
If the string must change, the following option has already been translated **62** times:
`translate( 'Sharing' )` *ES Score: 14*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/sharing-ellipsis)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).